### ISSUE TITLE
TreeSelect: Selected tags not showing up when used in dropdown menu

### ISSUE THREAD

**Issue by cjwang18** [NG-ZORRO/ng-zorro-antd#5625]
### Reproduction link
[https://stackblitz.com/edit/ng-zorro-antd-ivy-mmkvdd](https://stackblitz.com/edit/ng-zorro-antd-ivy-mmkvdd)

### Steps to reproduce
1. Click "Filter Users" to open up the dropdown menu
2. Click the tree select and select any person, e.g., "Person A"
3. Click outside to close the tree select dropdown. At this point, the selected tag is accurate (should show "Person A")
4. Click outside to close the dropdown menu
5. Click "Filter Users" to open up the dropdown menu again
6. Observe that the selected tag is not shown. However, if you inspect elements, the corresponding `nz-select-item` exists in the DOM.
7. Repeat steps 2-6 with
  a. unselecting the user you selected above leads to console error
  b. selecting a user on the same team leads to same error as above
  c. selecting a user on a different team leads to the issue stacking

### What is expected?
When showing the NzTreeSelect within a dropdown menu, the selected tags should persist and show correctly given an initial FormControl value.

### What is actually happening?
When there is an initial FormControl value, the NzTreeSelect doesn't render the selected tags properly on subsequent shows of the NzTreeSelect.

__After Step 6__
The NzTreeSelect dropdown is open to show that the nodes correspond correctly to the FormControl.
![image](https://user-images.githubusercontent.com/7762383/88730768-c1784c80-d0eb-11ea-9846-41b10c63631c.png)

__After Step 7c__
Notice the gap where the "Person A" tag would be.
![image](https://user-images.githubusercontent.com/7762383/88730580-3f882380-d0eb-11ea-9e48-4b5f2291b51c.png)

The console error mentioned above is:
```
ERROR TypeError: Cannot read property 'zoomMotion' of undefined
    at eval (transition_animation_engine.ts:383)
    at Array.forEach (<anonymous>)
    at AnimationTransitionNamespace.prepareLeaveAnimationListeners (transition_animation_engine.ts:375)
    at AnimationTransitionNamespace.removeNode (transition_animation_engine.ts:440)
    at TransitionAnimationEngine.removeNode (transition_animation_engine.ts:740)
    at InjectableAnimationEngine.AnimationEngine.onRemove (animation_engine_next.ts:70)
    at AnimationRenderer.BaseAnimationRenderer.removeChild (animation_renderer.ts:175)
    at DebugRenderer2.removeChild (services.ts:772)
    at execRenderNodeAction (util.ts:369)
    at visitRenderNode (util.ts:334)
```
It seems like references to previous `nz-select-item`s are lost.

| Environment | Info |
|---|---|
| ng-zorro-antd | 9.3.0 |
| Browser | Google Chrome 84.0.4147.89 (Official Build) (64-bit) |

**Comment by vthinkxie** [comment]
Duplicate of https://github.com/NG-ZORRO/ng-zorro-antd/issues/1455
blocked by https://github.com/angular/angular/issues/24041
